tests/unit_tests/rng/include/engines_api_tests.hpp

```diff
@@ -55,10 +68,10 @@ class engines_constructors_test {
             cl::sycl::buffer<std::uint32_t, 1> r3_buffer(r3.data(), r3.size());
             cl::sycl::buffer<std::uint32_t, 1> r4_buffer(r4.data(), r4.size());
 
-            oneapi::mkl::rng::generate(distr, engine1, N_GEN, r1_buffer);
-            oneapi::mkl::rng::generate(distr, engine2, N_GEN, r2_buffer);
-            oneapi::mkl::rng::generate(distr, engine3, N_GEN, r3_buffer);
-            oneapi::mkl::rng::generate(distr, engine4, N_GEN, r4_buffer);
+            generate(distr, engine1, N_GEN, r1_buffer);
```

Comment:
Why do you think that fully qualified name is not needed here?

From https://docs.microsoft.com/en-us/cpp/cpp/namespaces-cpp:
> Code in header files should always use the fully qualified namespace name.

Of course, the remark is not critical since the code is for tests

Reply:
This function isn't from this namespace. I defined it above. But I see that it can be a bit confusing, so I renamed it.